src/drivers/dw-dma.c

```diff
@@ -88,6 +88,8 @@
 	(0x0040 + BYT_CHAN_OFFSET(chan))
 #define DW_CFG_HIGH(chan) \
 	(0x0044 + BYT_CHAN_OFFSET(chan))
+#define DW_DSR(chan) \
```

Comment:
This register may not be available on all versions of IP, I guess you have only tested on SUE ?

Reply:
yes, but it is only used if the new .scatter flag is enabled, and so far it is only enabled on Sue Creek / by the SPI driver, so, at least this won't cause runtime problems. But if this register isn't available on some platforms, we *might* make it conditional on some preprocessor macro, that would make the code uglier though...

Reply:
SUE has the same IP as CNL, ICL for DW-DMA sou you could use the existing preprocessor macros.